### PR TITLE
220 make it possible to associate the hcl report with a given device and present what is in it

### PR DESCRIPTION
**Pull request by esesyn01**

### CODE REVIEW

File: meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report [hunk 1]
```diff
@@ -127,9 +156,17 @@ printf '################################################################    \r'
 printf '##################################################################  \r'
 printf '####################################################################\r'
 
-filename="$(dmidecode -s system-manufacturer)"
-filename+=" $(dmidecode -s system-product-name)"
-filename+=" $(dmidecode -s bios-version)"
+#filename="$(dmidecode -s system-manufacturer)"
+filename="$(dmidecode -s system-product-name)"
+#filename+=" $(dmidecode -s bios-version)"
```
Comment (kotylamichal) [inline meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report:161]:
```suggestion
```

File: meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report [hunk 2]
```diff
@@ -1,5 +1,34 @@
 #!/usr/bin/env bash
 
+print_information()
```
Comment (TomaszAIR) [inline meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report:3]: @esesyn01 @kotylamichal we would also like here to have informations if the logs are empty/with errors/clean. We could use functions similar to https://github.com/Dasharo/meta-dts-ce/blob/init-ce-dell-deploy/meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy#L8 to inform user what was dumped

Comment (kotylamichal) [reply meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report]: @TomaszAIR done https://github.com/Dasharo/meta-dts-ce/pull/9/commits/3c0e875398d082074092ca8f3f47cab46d14d1e3

**Review by TomaszAIR — APPROVED** [review]
Added some improvements but other than that looks good

**Pull request merged**